.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,44 @@
+<!--
+IF YOU DON'T FILL OUT THE FOLLOWING INFORMATION WE MIGHT CLOSE YOUR ISSUE WITHOUT INVESTIGATING
+-->
+
+**I'm submitting a ...**  (check one with "x")
+```
+[ ] bug report => search github for a similar issue or PR before submitting
```

Comment:
I think "have you searched for an existing, related issue yet?" as a statement emphasised at the top would be best instead of putting it down here. This can be applied to all issue types.